app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ActiveRecord::Base
+  has_many :posts, class_name: 'Post', foreign_key: 'author_id'
+  has_many :comments, class_name: 'Comment', foreign_key: 'user_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'user_id'
```

Comment:
- [ ] After getting done with the previous requirement, please modify those 2 relations to be `foreign_key: 'author_id'` 🙏